dashboard_api/db/static/sites/__init__.py

```diff
@@ -1,54 +1,73 @@
 """ dashboard_api static sites """
 import os
-from enum import Enum
-from typing import List
+import json
 
-from dashboard_api.db.static.errors import InvalidIdentifier
-from dashboard_api.db.utils import get_indicators, indicator_exists, indicator_folders
-from dashboard_api.models.static import Site, Sites
-
-data_dir = os.path.join(os.path.dirname(__file__))
+import botocore
+from cachetools import TTLCache
 
+from dashboard_api.db.utils import s3_get
+from dashboard_api.models.static import Sites, Link
+from dashboard_api.core.config import (SITE_METADATA_FILENAME, BUCKET)
+from dashboard_api.db.utils import indicator_exists, indicator_folders
+from dashboard_api.models.static import Site, Sites
 
 class SiteManager(object):
     """Default Site holder."""
 
     def __init__(self):
-        """Load all datasets in a dict."""
-        site_ids = [
-            os.path.splitext(f)[0] for f in os.listdir(data_dir) if f.endswith(".json")
-        ]
-
-        self._data = {
-            site: Site.parse_file(os.path.join(data_dir, f"{site}.json"))
-            for site in site_ids
-        }
+        self.sites_cache = TTLCache(1, 60)
```

Comment:
site manager uses a TTL cache of size 1 to store the entire Sites configuration in memory. The 60s timeout means it will be automatically refreshed to pickup new changes to sites published to the S3 bucket.